src/protocols/rdp/channels/audio-input/audio-buffer.c

```diff
@@ -120,6 +121,11 @@ void guac_rdp_audio_buffer_begin(guac_rdp_audio_buffer* audio_buffer,
         void* data) {
 
     pthread_mutex_lock(&(audio_buffer->lock));
+    
+    /* Create a thread */
+    audio_buffer->stop_thread = 0;
+    pthread_create(&audio_buffer->send_thread, NULL,
+                   guac_rdp_audio_send_thread, audio_buffer);
```

Comment:
Please make sure that your comments provide more context than simply restating the code itself in English.